---
title: Phages Biology
date: 2022-06-28
author: Runhang
---

### Last up date: 04-21-2023

An understudied field of research has gained increasing attentions recently after scientists saw numerous viral particals in the soil, ocean, bacterial cells and ,etc by using fluorescence electron microscope or transmission electron microscope. The virues that infect bacteria are called bacterialphages or phages, mean "to eat" in Latin. It is now recognized as the most abundant biological entities (not an organism but a molecular machine) on the planet. So, the article will introduce what do we know about them, and what remains mysterious? 

**Lysogeny** is the phage genome nested in bacterial host chromosomes. One copy of phage genome will be made during each replication of a bacterium. Lysogeny phage, or prophage, is usually temperate phage that have a long-term relationship with bacteria. 

**Piggback-the-winner** is a strategy that prophages use to replicate together with the more competitive bacterial hosts. 

**Lysis** occurs when a type of more virulent phage replicates its genome and packaging bacterial genome inside bacterial cells, ending up lysing the cells and releasing all the "free" phages. This type of virulent phage is most studied in the lab than temperate behaviors of prophage in nature enviroments. 

**Kill-the-winner** is a strategy for virulent phages who are typically obligately lytic, killing their hosts after infection. 

**Superinfection exclusion** is an outcome of lysogeny, where the prophage makes the host immune aginst infection by closely related or unrelated phages. 

1. The mechanisms underlying superinfection exclusion are diverse. First, it can be modifications of the surface receptor (i.e., type IV pilus and O-antigen) before the virus entering the cell. Moreover, the phage-resistant mechanisms of lysogen could occur intracellularly after the phage DNA is injected into the cell. 
2. In PA14, it is an addictive effect that bearing multiple phages usually leads to more broad immunity to different phages. 
3. In PA14, another study showed that prophage-encoded protein, *Tip*, blocks the activity of bacterial-encoded PilB, rendering superinfection exclusion and defects in twitching motility ([Chung et al., 2014](https://www.pnas.org/doi/abs/10.1073/pnas.1403537111)). 

**Lysogenic to lytic conversion** can be achieved under certain stresses (DNA damage, py)when CI or CI-like repressor protein is inactivated via the SOS-response (in the case of E. coli lamda phage). For instance, Genotoxic agens like mitomycin C triggers E. coli SOS-response through DNA damage, resulting in upregulated recombinase gene expression such as RecA. The will lead to clevage of CI repressor and activate lytic pathway. The first step in lytic induction is prophage excision, which requires both phage-encoded integrase and excisionase. The lysogenic conversion only requires integrase. 

In the rare case such as Mu phage, it might not be excised during induction. Instead, the Mu prophage can get duplicated and moved to a new location during induction. It acts like a transposable element or we could call it transposable phage. 

**Phage can directly modulate host bacterial physiology or direct bacterial fate**

1. The phage genome bears many AMGs (Auxiliary metabolic genes) that can participate in host metabolic pathways. 

2. The interplay between nutrient demand and availability and the plasticity of the metabolic state of "virocell" may influence the lysis-lysogeny decision. For example, during spring-summer seasonal transitions in Southern Ocean, temperate virues switch from lysogeny to lytic replication as bacterial production increases (Brum et al., ISME, 2016). However, the virus to bacterium ratio remained largely unchanged. In contrast, another study on coral reefs suggest "Piggyback-the-Winner(PtW)" hypothesis, by demonstrating that increased host density is accompanied by a transition from lytic to temperate dynamics (Knowles et al., Nature, 2016). In general, oceans are "prokaryotic" world with virus outnumbering bacterium by 10:1, indicating a high rate of lysis, whereas an "eukaryotic" world such as human gut has virus to bacterium ratio of 1:1, indicating many phages exist as an inducible prophage form (Reyes et al., 2012). Well, I think Knowles' paper from Forest Rohwer lab is more broadly applied to many ecosystems such as human gut virom, where microbial density is high compared to ocean samples. Such a high density of microbes is consistent with PtW hypothesis as lysogenic conversion is more prevalent in the human gut. The cell densities is highest in coast followed by surface ocean ,and deep ocean has the lowest cell density. The bacterium-phage symbiosis plays different strategies among these ecosystems. Silveira et al., (2021) proposed a framework wherein phage integration occurs more frequently at both ends of the host density gradient, with distinct underlying molecular mechanisms (coinfections and host metabolism) dominating at each extreme.

3. Virus-encoded tRNA can also serve to enrich the level of tRNAs that match the viral codon usage. 
4. One of the most famous example is CTXphi phage makes _Vibrio cholerae_ produce cholera toxin (2005). 
5. Some PA14 strains lysogenized by DMS3 are unable to form a biofilm or undergo swimming mobility. 

**Phage infections lower the global transcription but not protein abundances of bacterial cells** (https://www.nature.com/articles/s41396-019-0580-z). 

1. Such effects vary greatly among phages. The two Pseudoalteromonas phages, HP1 and HS3, used in this study share only 1% of their genome. Both of them can infect the same host effectively. 

2. Fitness is different. 

3. The host-phage codon dissimilarity can reflect extent of the metabolic demand to synthesize proteins for phages. A high host-phage codon dissimilarity is considered as reprogramming the host transcription to meet the nutritional needs of phage infection/replication. In contrast, a low host-phage codon dissimilarity is indicative of the virocell providing intracellular environment and resources needed for infection sucess. They are two different strategies. 

4. Phages with little complementarity need to reprogram cellular metabolism prior to lysis. The burst size is smaller than a phage that has greater complementarility with host. That is to say the later one is putatively "temperate" and can co-exist with host for a relatively longer time before burst. Alternatively, it can become a prophage via lysogeny. 

**Counting viral particles is not equivalent to counting virues - Patrick Forterre**

1. Therefore, a commonly wrong statement is that viruses outnumber cells in the environment by 10 times. How can virocell (the phage-infected cells)  are more abundant than visible cells when they are physiologically a subpopulation of cells in the enviroment. 

2. This does not reduce the importance of viruses as viral genomes greatly outnumber cellular genomes in the biosphere. 

**Up to 40% of bacteria present in bacterioplankton are infected by viruses (Suttle, 2007).**

**It is intriguing that most genes in viral genomes have no cellular homologues and only a small percentage can be traced to cellular ancestors.**

1. In fact, the integration of viral genes into cellular genomes is probably more frequent than steel genes from the host. 

2. One can claim that cells are gaint pickpockets of viral genes, not vice versa (Patrick Forterre 2012, *ISME J*). 

3. In metagenomic data analyses, it is a challenging work to distinguish cellular genes from viral genes. To build a complementary database with viral proteins is the first step. This database is a reservoir of biologically information stored in viral genomes. These are orphan genes with no homologues in current datebases or viral-specific genes. The virocells are considered as cradles of new genes. 

**Bacterial cell surface has receptor for bacteriophages, such as LamB of *E. coli*, serves as an entry point by phage lamda.**

1. The deletion of *LamB* in *E. coli* will abolish superinfection (phage infection of already phage-infected bacterium or lysogeny). This is one explaination of more free phage in the bacterial culture because lysogenys become immune to superinfection.

2. Meanwhile, the lytic process is still happening, leading to accumulated free phage in the culture. 

3. Hoyland-kroghsbo et al. (2013) revealed that N-acyl-l-homoserine lactone (AHL), a quorum-sensing signal, supresses λ receptors on the cell surface (measured by SDS-page). As a result, the AHL-induced quorum-sensing activates host immune defense to phage superinfection. (DOI:https://doi.org/10.1128/mBio.00362-12)

**Bacteria employ multiple mechanisms to defense against phage infection**

We can broardly divide bacterial defense mechanisms into two aspects. One is constitutive defense (always active) such as the  the surface receptor modidfication/masking. Another is inducible defenses (inducible by virus) such as CRISPR. Cell surface is probably one of the most important mechanisms because phages will exert damage to the bacterial cell once entered despite a timely recruitment of CRISPR. The constitutive defense is often energetic costly when there is no phage around while the adaptative CRISPR immunity might be more dynamic to adjust how many spacer the genome wants to keep. 

1. CRISPR-Cas immune system. Only 50% bacteria have this mechanism in their genome. ([Goldberg et al., 2018](https://www.nature.com/articles/s41467-017-02557-2)) How does CRISPR differenciate the "non-self" prophage from "non-self" lytic phage? The CRISPR-cas has a transcription-dependent DNA targeting mechanism that recognizes the lytic cycle-related transcripts but permits lysogenization by temperal phages. The so called "conditional tolerance" of temperal phage is resumed upon lytic induction ([Goldberg et al., 2014](https://www.nature.com/articles/nature13637)). 
2. Down-regulating phage receptor. Serve as a first line of defense. 
3. At high cell density, Pseudomonas QS activates cas3 expression. The QS-mediated CRISPR-cas immune system serves as a second line of defense after phages have entered into cytoplasm ([Nina et al., 2017](https://www.pnas.org/doi/epdf/10.1073/pnas.1617415113)). 
4. Joseph at UCSF recently published some noval phage defense mechanisms on P. aeruginosa. 

**Bacteria-virus have been co-evolving for billions of years, so prophages/phages have acquired a collection of strategies to defend against superinfection and bacterial innate immunity 

1. Bacteriophages cooperate to deploy anti-CRISPR protein to suppress CRISPR-Cas3 and Cas9 immunity. (https://www.cell.com/cell/pdf/S0092-8674(18)30738-4.pdf)
2. [Surface modification](https://www.nature.com/articles/ismej201679#Sec1) is very common in *P. aeruginosa*, where the prophage affects T4P and O-antigen to resist superinfection. Because phage predation is a major threat to bacterial survival, this mechanism allows losogenized PA14 to resist multuple different phages in addition to the self phages. 


**Some phages suppress host QS-mediated defense**

1. Phage DMS3 encodes a quorum-sensing anti-activator protein, Aqs1, that inhibits LasR ([Shah et al., 2021](https://doi.org/10.1016/j.molcel.2020.12.011)). The supressed QS allows higher rates of lysogenization. 
2. Other than host defense mechanism, a prophage can produce superinfection exclusion protein to prevent being replaced by same or closely related phages. 



**Factors affect phage-mediated cell fate ([Zheng et al., 2011](https://doi.org/10.1016/j.cell.2010.03.034))** 

1. The presence of absence of prophage. The lysogeny is immune to the phage infection. 
2. Cell size 
3. MOI (multiplicity of infection, the number of phages infecting an individual cell. A high level of MOI (>10) or API (average phage input) can result in 100% lysogens. However, it is phage-dependent. 
4. Location (pole, mid-cell, which contains key protein such as ManY for DNA injection)
5. Lysogeny requires that all infecting phages choose lysogeny. The logic of the cellular decision can be thought of as a simple “AND” gate, such that only if all inputs are “1” (i.e., lysogeny) will this be the cellular output.

The reason for the inadequacy of single-cell resolution is that the cell-fate decision is achieved through a hierarchy of decisions at the sub-cellular level. *Bacillus subtillis* SPβ phage (strain phi3T) encodes AimP, which processed into six amoni acids (aa) peptides for downstreading decision of either lysogenic or lytic cycle. This 6-aa communication system is also known as arbitrium system first described by Dr. Rotem Sorek at Weizmann institute, which is essential for phage to communicate and decide to "break" the cell or "hide" in the cell. For example, if the abundance of 6-aa is too high, the phages would interpret that they are running low on uninfected cells and thus the circuit will regulate phage from lytic to lysogenic cycle. Therefore, bacterial hosts will not be completely depleted.  

**Teachniques of determining whether cells are going through lysis or lysogenic process.([Trinh et al., 2017](https://doi.org/10.1038/ncomms14341))**


1. Label phage with YFP by fusing YFP with wild-type head-stabilization protein gpD located at viral capsid (gpD-YFP). Therefore, if cells are lysed, we will see yellow fluoresence under a microscope. On the other hand, the lysogenic pathway is indicated by the production of mCherry fused with P<sub>RE</sub> promoter. Lytic pathway does not exhibit P<sub>RE</sub> activity. Likewise, there are certain host genes participate in lytic pathway. For example, lambda replication is severely inhibited in dnaJ mutants of *E. coli*. 
2. Additionally, lysogenic reporters can be constructed by transcriptionally fusing mKO2 (yellow) and mKate2 (red) to the phage lytic repressor gene, cI. 
During lysis process, phages compete resources for DNA replication. During lysogenic process, phages cooperate to propagate for integration. 
 

### Experimental techniques in bacteriophage study 

**Isolating phage from bacteria**
To isolate phages from the culture, the bacterial cells from the overnight culture were lysed by the addition of a few drops of chloroform and subsequent shaking at 37C for 30 minutes. Cellular debris was collected by centrifugation at 15000 x g for 10 minutes. The phage lysate was stored at 4C with a few drops of chloroform to keep it sterile (Shah et al., 2021).
 
**Plaquing assay**
To assess the sensitivity of bacteria to various phages, serial dilutions of phages were spotted onto lawns of bacteria in the plaquing assay. To create a lawn of bacteria, 150 uL of overnight bacterial culture was added to 3 mL of 0.7% molten top agar and poured atop 1.5% LB agar plates. Both the top agar and bottom plate were supplemented with 10 mM MgSO4 and 50 mg/ml of gentamicin and 0.1% L-arabinose where required. 2mL of 10-fold serial dilutions of phages were spotted on to the bacterial lawns, and the plates were incubated at 30C overnight (Shah et al., 2021).
  
- temperal phage will form turbid plaque on the bacterial lawn. 
- lytic phage will form clear plaque on the bacterial lawm. 

**Why?** There is a good question and answers in this [Researchgate thread](https://www.researchgate.net/post/How-to-explain-the-turbid-and-halo-plaques-formed-by-some-bacteriophages#:~:text=Turbid%20plaques%20are%20usually%20produced,plaque%20will%20look%20'turbid'.). 

**Question:** For the clear plaques, I can understand it's a complete lysis, no bacteria are growing in that aera any more. But how to explain the turbid plaques? Is it the phage lysed some of the bacteria, but the rest got resistant to the phage, which led to a incomplete lysis? And how about the halo plaques?
And why the plaques have certain size, but cannot enlarge limitless?

**Answer** by Alejandro Martin: It depends a bit on the life cycle of the specific phage you are dealing with. Turbid plaques are usually produced by lysogenic phage such as lambda. In some of the cells the phage may lysogenize instead of continuing the lytic cycle, and if this happens with high enough frequency the plaque will look 'turbid'. Other times (I am guessing this is what you call 'halo plaques') phage replication does not lyse the cell, but slows their growth noticeably enough that you can distinguish a plaque in solid medium; a typical example are filamentous phages. As for why phage plaques do not grow to lyse the entire plate, one of the reasons is that most phages cannot productively infect stationary phase cells, so once the bacterial lawn reaches that stage it's game over for the phage.

**Check if the bacterium is a lysogen**. Isolate cells (by limiting dilution, streaking or whatever) from the turbid area of the plaque and check them for 1)resistance to infection by cross-steaking assay ;2) check the presence of phage in the lysate; 3) check integrated phage genome by PCR. A lysogen is immune to the same phage so that the phage will not form plaques. 
 
*Apart from using morphology to roughly determine if the phage is lytic or lysogenic (temperal), how does sequencing work?*

- We can design phage primers to run a PCR. Need to re-streak a few times and to make sure you don't have any contaminating phage along for the ride that would also give you PCR product.
- But if we do not know the phage sequence or want to know where does it insert at the beginning, we can do a whole genome sequencing. 

**There is exceptions of forming turbid plaques, which is independent on lysogen**

For instance, the turbidity might arise also from spontaneous mutation or antigenic phase variation of the initial receptor used for phage attachment/entry, but these are less frequent events.

**When filamentous phage virions (pf phages) are produced, they are generally extruded without bacterial lysis **

**How to generate prophage-free strain from a lysogen?**

By using allelic exchange, [Rice et al., (2009)](https://www.nature.com/articles/ismej2008109) knocked out the entire filamentous Pf4 prophage from PAO1 using a suicide plasmid. 

This method was first applied in the mid 1980s, and read more [here.](https://doi.org/10.1186/1471-2180-5-30) and [here] (https://www.nature.com/articles/nprot.2015.115). 

**Research directions in phage study**

1. the role of phage in host bacterial pathogen short-term adaptation 
2. phage therapy (phage cocktail to treat ARB)
3. microbial and phage communities 
4. phage pathogenesis: promote host virulence. For example, shiga, diphtheria and cholera toxins which greatly enhance virulence of *E. coli*, *C. diphtheria* and *V. cholera* infections, respectively, are all encoded by morons (prophage-associated genes).

**Phages are involved in the bacterial immune system to host-induced stress**

1. The Gifsy-2 Salmonella prophage encodes the superoxide dismutase (SOD) that protects the bacterium against free radicals generated by the mammalian immune system
2. Some phage genes antagonize host serum killing, degrade host antibidies (IgG), disrupt host immune reaction chain, and are thus ineffective in clearing the bacterial infection. 
3. Some phages help the bacteria evade host cell recognition by acetylating the O-antigen of bacterial lipopolysaccharide. 

## Phage insertion site

**Insert at a specific site**
Phages typically reply on their integrase to mediate a recombination event between an identical sequence shared between the circular form of the prophage genome (*attP*) and the bacterial chromosome (*attB*), and the recognition of these DNA sequences is inherent in a given integrase protein. 
**Insert at random site**

## Phages genome size and their lifestyle

Do temperate phages tend to have a smaller genome as they have to streamline their genome as they integrated into the bacterial chromosome, in contrast to lytic (virulent) phages? On the other hand, we conceptualize temperate phages as USB drivers, that transfer information between bacterial kin. Therefore, one can argue, temperate phages should have a larger genome size than lytic phages. Let's take a look into some public datasets and dig this out!  

![Refseq_phages](https://github.com/RunhangShu/RunhangWebsite/assets/45881840/9d62b771-0e48-4757-a04b-b1c6b9b03fe7)
I analyzed all phage genomes curated by NCBI-RefSeq and found virulent phages have a medium size of ~80kbp, followed by 53kb of temperate phages. Finally, chronic phages such as Pseudomonas filamentous phage pf5, recently characterized as a standalone lifestyle differ from lytic/lysis as these "chronic phages" replicate in host cells and extrude out without causing cell lysis, 


![across_genus](https://github.com/RunhangShu/RunhangWebsite/assets/45881840/31c7902a-ffbe-45b2-bb6f-4d5e0ba755fe)

The data used to generate the figure above is from phage.ai. 

### Ecological dynamics of microbiome and bacteriophage

1. Bacteria-phage interactions can be quite different depending on the context. Many assumptions about human microbiome-phage interactions are originally drawn from marine virome research ([Shkoporov et al., 2022](https://www.nature.com/articles/s41579-022-00755-4)). This leads to increasing studies on investigating the possibility of leveraging phages to kill pathogenic bacteria. Is their relationship so simple? Is mutualistic relationship common in the context of human gut microbiome-phages? I guess the big question is how permissive that a bacterium allow the phage to take advantage from it. How does bacterium balance the lytic process (sacriface for a good outcome) and lysogenic process ("uneasy truce", keep the phage genes for a potential cost). 

2. Taking a snapshot of phagenome or virome in human gut is a taxonomically difficult job. Sequencing reads show that 85%-99% of viruies are novel without any classification and host information. 
- **Huge VARIATION**: The interpersonal variations are dramatic that very few viruses shared between unrelated individuals. 
- **High STABILITY**: longitudinal metagenomics data showed that 80% virual clusters (grouped by 90% identity 90% sequence identity over 90% of contig length) is still present even after 2.5 years. 

3. A large fraction of the gut virome is composed of virulent or lytic phage. So, the stability is not majorly due to integrated viral genome.
4. Phage might play a minor role in serving as a vehicle in HGT. For instance, ARGs (antibiotic resistant genes) are rarely found in the phagenome. 


## Are phages inserted into bacterial genome randomly? Do they have a preferred insertion site?

1. The DMS3 phage in *P. aeruginosa* does not have a preferred insertion site. The DMS3 genome was found in multiple loci in *P. aeruginosa*. This is same as the Mu phage of E. coli.  

## Phages in the context of a microbial community

1. While many temperal phages are integrated into bacterial genome, many are found as free phage virions embedded within the mucus layer and have been proposed to function to maintain the intestinal barrier by controlling invasive bacterial populations (Barr et al., PNAS 2013). 
2. The most notable change is an increase in the order of bacteriophages, Caudovirales, within the intestine of individuals with Crohn’s disease (CD) and ulcerative colitis (UC). 


## Ideas of my own project

1. When does a lysogen decide to lyse or to main the status quo? Or, I guess, it is more appropriate that we think this question from the perspective of phage; when does the prophage promote induction? 
- when the surrouding nutrition is low - bacterial host is starving. How??
- when there is target-rich environment. How??

2. What are the molecular mechanisms underlying such delicate regulations of lytic and lysogenic pathways? 
3. Pf phages are recently repored to directly impede mammalian phagocytosis by targeting both neutrophils and macrophages ([SWEERE et al., 2019](https://www.science.org/doi/full/10.1126/science.aat9691)), allowing its bacterial host to survive. Understanding the mechanisms of phage-mediated mammalian immune system is critical for the advance in phage therapy. 
4. 

## What are more than lytic/temperate phages? How about other genetic elements stem from phage? 

### 1. Phage-related chromosome islands (PRCIs) have been found in Gram-positive bacteria such as Staphylococcus aureus.

- It is a phage-related mobile genetic elements, which could evolved from a prophage. 
- Usually quiescent and requires a helper phage to abolish the repressor Stl.
- It can hijack the helper phage's capsid to propogate. 
- High-frequency transfer of SaPIs (S.aureus pathogenicity islands) was demonstrated. 
### 2. Transposable element 
- I feel like TEs is basically temperate phages which lost structure genes?

### 3. Plamid-dependent phages, Sian Owan 

### 4. Phage regulatory switch (phage-RS) system: Both DNA and protein sequences suggest they are remnants of temperate phages 

In *B. subtilis skin*-interventing DNA element, which is a 48kb remnant of an ancestral phage. It can not produce infective viral particles, but it retained its ability to excise itself to restore an intact functional gene flanking to this element. This step is achieved by specific recombinase and other accessory proteins inside the phage-RS element. This gene rearrangement gives increased fitness to the bugs under special conditions (Sato et al., Plos genetics, 2014). 
Another example about phage-RS is in S. pyogenes, where the SpyCIM1 is maintained as an episome floating around in the cytosol. Under stress, however, the phage will insert right in the middle of mutsL, which render the mismatch repair (MMR) system non-functional, leading to over 100-fold increase in mutation rate.  
Different from DMS3/Mu phage, each phage-RS element is inserted in the same location within its cognate gene. 

## Phage serves as a mutator

### Mu phage
- Mu phage was discovered in late 1950s. It inserts into bacterial genome randomly, which resembles the "controlling elements" of maize more closely than any other bacterial episome
- On rare cases, it will excise itself from the genome to restore orginal function of the operon at that inserting site. However, it also copies itselft and inserted into a new site without leaving its original location. (It is worth to see if our lysogens have mutiple DMS3 prophages) Therefore, it is a transposible phages using the replicative transposition strategy. 
- The very early identified transposition mechanisms for Mu is known as Shapiro model, which is the first example of non-homologous, strands joining, yet somewhat site-specific recombination. It has been 30 years after this first model, and many more mechanisms of transposition have been revealed. 

## Professors in phage study 

1. Carey Nadell @Dartmouth. [Recent talk in 2022](https://www.youtube.com/watch?v=Z6HvLuzZuMY&t=1232s)
2. George O'toole @Dartmouth.
3. Paul Turner @Yale 
4. Joe Bondy-Denomy @UCSF
5. Alan Davidson @Univ. of Toronto
6. Karen L Maxwell @Univ. of Toroto 
7. Paul Bollyky @Standford. https://www.science.org/doi/full/10.1126/science.aat9691. 
8. Breck Duerkop @University of Colorado (enterococcus resistance to phage)
9. Rasika M Harshey @University of Texas at Austin (Mu phage, transposition, Esi.coli swarming) https://sites.cns.utexas.edu/harsheylab/bacteriophage-mu
10. Jeffrey Gordan @WUSTL
11. Forest Rohwer @SDSU
12. Jilian F.Banfield @UC, Berkeley ([CRISPR could be less common that previously reported](https://www.nature.com/articles/ncomms10613); sequenced and found hundreds of gigantic, CRISPR-Cas system-carrying phages with lengths of more than 200 kb (https://www.nature.com/articles/s41586-020-2007-4) across different ecosystems. 
13. Travis J Wiles @UCI
14. 

